docs/best-practices/linux-windows.md

```diff
@@ -0,0 +1,103 @@
+# Using _fastlane_ on Linux and Windows
+
+Fastlane was created to automate building iOS apps on macOS. Over time, support for building macOS 
```

Comment:
Lower case fastlane

Reply:
(New and slightly hidden GitHub feature: First icon in the toolbar lets you add suggestions on PRs that people can 1-click accept)

```suggestion
fastlane was created to automate building iOS apps on macOS. Over time, support for building macOS 
```